core/toxidcurl.php

```diff
@@ -114,6 +114,8 @@ class toxidCurl
      */
     private $additionalUrlParams = array();
 
+    private $notFoundUrl = 'de/startseite/404-not-found/';
```

Comment:
Hardcoded Url are not allowed in TOXID module.
This is a generic open source module.